demo/src/app/components/+dropdown/demos/dropup/dropup.ts

```diff
@@ -1,9 +0,0 @@
-import { Component } from '@angular/core';
```

Comment:
why did you remove dropup demo?

Reply:
cause I've added it in another pr (docs(dropdown): decompose demo for dropup property #3790)